src/TimeService.Business/Helpers/Workdays/WorkTimeLimitCreater.cs

```diff
@@ -0,0 +1,105 @@
+﻿using LT.DigitalOffice.TimeService.Data.Interfaces;
+using LT.DigitalOffice.TimeService.Models.Db;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.TimeService.Business.Helpers.Workdays
+{
+    public class WorkTimeLimitCreater
+    {
+        private readonly IWorkTimeMonthLimitRepository _limitRepository;
+        private readonly WorkCalendarHelper _calendarHelper;
+        private readonly ILogger<WorkTimeLimitCreater> _logger;
+
+        private int _minutesToRestart;
+        private DateTime _lastSuccessfulAttempt;
+        private DateTime _previousAttempt;
+
+        private string GetNonWorkingDays(DateTime time)
+        {
+            const string logMessage = "Cannot get non-working days.";
```

Comment:
this string using only in catch. declare it there

Reply:
fixed